charts/templates/karmada_apiserver.yaml

```diff
@@ -70,6 +70,8 @@ spec:
             - --requestheader-username-headers=X-Remote-User
             - --tls-cert-file=/etc/kubernetes/pki/karmada.crt
             - --tls-private-key-file=/etc/kubernetes/pki/karmada.key
+            - --max-requests-inflight=1500
+            - --max-mutating-requests-inflight=500
```

Comment:
Just a question, can users configure this parameter?

leave to owners @jrkeen @algebra2k

Reply:
ok,we can have a default value,then make them configurable

Reply:
> ok,we can have a default value,then make them configurable

I agree with you